code/modules/admin/admin_verbs.dm

```diff
@@ -880,32 +880,25 @@ var/list/admin_verbs_mod = list(
 			src << "You are already an admin."
 			verbs -= /client/proc/readmin
 			return
-		var/sql_ckey = sanitizeSQL(ckey)
+		var/sql_ckey = sanitizeSQL(ckey((ckey)))
 		var/DBQuery/query = dbcon.NewQuery("SELECT ckey, rank, level, flags FROM erro_admin WHERE ckey = [sql_ckey]")
 		query.Execute()
-		usr << "Query executed"
 		while(query.NextRow())
 			var/ckey = query.item[1]

```

Comment:
I do when figuring out what broke. It showed me the query gave no results but it did go through successfully